helm/cluster-api-ipam-provider-in-cluster/templates/pools/pool.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.inClusterIpPool.enabled }}
+apiVersion: ipam.cluster.x-k8s.io/v1alpha1
+kind: InClusterIPPool
+metadata:
+  name: {{ .Values.inClusterIpPool.name }}
+spec:
+  gateway: 0.0.0.0
+  prefix: 24
+{{- with .Values.inClusterIpPool.start }}
+  start: {{ . }}
+{{- end }}
+{{- with .Values.inClusterIpPool.end }}
+  end: {{ . }}
+{{- end }}
+{{- with .Values.inClusterIpPool.addresses }}
+  addresses:
+  {{- range . }}
+  - {{ . }}
+  {{- end }}
+{{- end }}
+{{- end }}
```

Comment:
Feels like equivalent of `{{ .Values.inClusterIpPool | toYaml | nindent 2}}`

Reply:
you are right, I can simplify this

Reply:
I reworked it a little bit, but `{{ .Values.inClusterIpPool | toYaml | nindent 2}}` can't be used w/o changing the structure of values, which I don't like